src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int base;
+    private final int height;
+
+    public IsoscelesTrapezoid(String color, int base) {
+        super(color);
+        this.base = base;
+        this.height = (int) ((base * Math.sqrt(3) / 2));
+    }
+
+    @Override
+    public double area() {
+        return 0.5 * base * height;
+    }
+
+    public void draw() {
+        System.out.println("Figure: trapezoid" + " area: " + area() + " firsBase: " + base
+                            + " units" + " secondBase: " + base + " units" + " height: " + height
+                            + " units" + " color: " + color);
+    }
```

Comment:
The draw() method should not be responsible for printing the figure details. Consider separating the responsibilities: one method to return the figure details as a String, and another to print it.